AimBot/AimBot.cpp

```diff
@@ -88,7 +88,15 @@ vector<float> Aim(Player entity[64], Player myplayer, int closest)
     }
 }
 
-HANDLE proces;
+void Attack()
+{
+    keybd_event(VK_F1, MapVirtualKey(VK_F1, 0), 0, 0);
+    Sleep(2);
+    keybd_event(VK_F1, MapVirtualKey(VK_F1, 0), 2, 0);
+    Sleep(20);
+}
+
+HANDLE process;
```

Comment:
You don't need a global variable, never. It can be declared in `main`. If you are using it in some method, just pass it as argument.